fractal-puppy/Engine
# Updated chest

Some rework to improve the chest:
- New animation.
- Spawned items are spawned with a jump animation.
- You need to click over the chest to open it.
- The chest is highlighted on mouse hover.

**To test:**
1. Open the App and load ItemsTest scene.
2. Hit play and reach the first chest (it is after the first door).
3. Check that a gold highlight is shown on mouseover.
4. Open the chest by clicking on it.
5. Check that the items are spawned with an animation.
**Example:**
https://gyazo.com/d928a29f6a09ba6158171f4660310993
